client/src/containers/example/index.tsx

```diff
@@ -81,4 +89,9 @@ const Example: React.FC = () => {
 	);
 };
 
-export default Example;
+function mapStateToProps(state: any) {
+	const { theme } = state;
+	return { theme };
+}
```

Comment:
I don't think this is important, but why do you use connect if you can use useSelector? Also, I thought we use connect function only in class components (not in React.FC), but maybe I am wrong.